python/py-GridDataFormats/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-GridDataFormats
+version             0.4.0
+revision            0
+categories-append   science
+platforms           darwin
+license             LGPL-3
+supported_archs     noarch
+
+python.versions     27 36 37
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         GridDataFormats provides the Python package 'gridData'.
+long_description    ${description} \
+                    It contains a class ('Grid') to handle data on a regular grid --- basically NumPy n-dimensional arrays. \
+                    It supports reading from and writing to some common formats (such as OpenDX).
+
+homepage            https://github.com/MDAnalysis/GridDataFormats
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  530f9dc78eefee6e551fd93377b25842d8de8a12 \
+                    sha256  f81d6b75aa7ebd9e8b64e14558c2d2583a0589829382beb4ef69860110261512 \
+                    size    63643
+
+if {${name} ne ${subport}} {
+    depends_build-append port:py${python.version}-setuptools
```

Comment:
whitespacing is incorrect here (as @mfk mentioned before); the right column should start at a multiple of 4 space (so not at 26, but either at 25 where you wrap the depends_build-append line or at 29).